src/components/CardComponent.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+<div class="card">
+    <div class="card-heading">
+        {{content.title}}
+    </div>
+    <div class="horizontal-line"></div>
+    <div class="card-main-content">
+        {{content.details}}
+    </div>
+    <div class="horizontal-line"></div>
+    <div class="likes" @click="likeBtnHandler">
+        <span class="heart-icon" :class="{red : content.liked}">&#10084;</span>
+        {{content.likes}}
+    </div>
+    <div class="hash-tags">
```

Comment:
@anubha-innotech this should be in the Hashtag component. As per doc
**Create a Hashtag component and render all hashtag-related views here.**